src/Microsoft.AspNet.Razor.Runtime/TagHelpers/HtmlTargetElementAttribute.cs

```diff
@@ -79,5 +79,11 @@ public HtmlTargetElementAttribute(string tag)
         /// </para>
         /// </remarks>
         public TagStructure TagStructure { get; set; }
+
+        /// <summary>
+        /// The required HTML element name of the direct parent.
+        /// </summary>
+        /// <remarks>A <c>null</c> value indicates any HTML element name is appropriate.</remarks>
```

Comment:
There's a couple of references to `string.IsNullOrWhitespace`. Do the checks apply to the value of this property? If so, the comment should say `A <c>null</c> or empty string ...`